devices.js

```diff
@@ -9208,8 +9208,8 @@ const devices = [
         vendor: 'Centralite',
         description: '3-Series security keypad',
         meta: {configureKey: 1, battery: {voltageToPercentage: '3V_2100'}},
-        fromZigbee: [fz.command_arm_with_transaction, fz.temperature, fz.battery, fz.ias_ace_occupancy_with_timeout],
-        exposes: [e.battery(), e.temperature(), e.occupancy(), e.action([
+        fromZigbee: [fz.command_arm_with_transaction, fz.temperature, fz.battery, fz.ias_ace_occupancy_with_timeout, fz.ias_zone_tamper],
```

Comment:
Instead of `ias_zone_tamper` the existing `ias_no_alarm` should also work.

Reply:
I tried it out but it gives me the following error:

>  No converter available for '3400-D' with cluster 'ssIasZone' and type 'commandStatusChangeNotification' and data '{"extendedstatus":0,"zonestatus":0}'

I guess it's because `ias_no_alarm` is of type `attributeReport`, but the data comes in as `commandStatusChangeNotification`?

Reply:
For the `ias_no_alarm` change `type: 'attributeReport',` to `type: ['attributeReport', 'commandStatusChangeNotification'],`, then it should work

Reply:
Ah makes sense.

I tried it out, but the state of the tamper field doesn't change using that method.

Reply:
Can you try changing `const zoneStatus = msg.data.zoneStatus;` to `const zoneStatus = msg.type === 'attributeReport' ? msg.data.zoneStatus : msg.data.zonestatus`

Reply:
@Koenkk sorry it took so long. That also doesn't work.